sklearn/manifold/_mds.py

```diff
@@ -274,6 +275,67 @@ def smacof(dissimilarities, metric=True, n_components=2, init=None, n_init=8,
         return best_pos, best_stress
 
 
+def svd(dissimilarities, n_components=2):
```

Comment:
hmm, I guess `svd` is not the best name here, cause it doesn't return the SVD really. Maybe `svd_scaler` or something? Not sure.

Reply:
Done